### PR TITLE
Add fonts

### PR DESCRIPTION
**Pull request by muffinresearch**
* Add Fira Sans webfonts (woff2, woff and otf) we don't need eot since we won't be worrying older IE versions.
* Fix clean script to remove everything from dist except the .gitkeep file.

### CODE REVIEW

**Comment by kumar303** [comment]
I see some binary files that appear to be unused in the `scss`, like `UltraLightItalic` and `ExtraBold`, etc. Do we still need to include them?

File: webpack.dev.config.babel.js [hunk 1]
```diff
@@ -61,27 +61,38 @@ export default Object.assign({}, webpackConfig, {
...
-      test: /\.jpg$/,
-      loader: 'url?limit=10000&mimetype=image/jpeg',
-    }, {
-      test: /\.webm$/,
-      loader: 'url?limit=10000&mimetype=video/webm',
-    }, {
-      test: /\.mp4$/,
-      loader: 'url?limit=10000&mimetype=video/mp4',
-    }],
+    loaders: [
```
Comment (kumar303) [inline webpack.dev.config.babel.js]: This looks similar to what's in the karma config. Is there an easy way to combine them?

**Comment by kumar303** [comment]
I don't know much about fonts but r+wc :)

File: webpack.dev.config.babel.js (hunk 1 above)
Comment (muffinresearch) [inline webpack.dev.config.babel.js]: Yep, that's a separate yak to shave issue is filed here: https://github.com/mozilla/addons-frontend/issues/353

**Comment by muffinresearch** [comment]
I'll leave the extra fonts for now, it's early days so we don't really know what we might need.

**Pull request merged**